Maven test step fails when stray xml files are present in hidden directories

## Describe the bug
The maven build fails when there are "stray" xml files present in the source code directory-tree, even if they are unrelated to DSpace. 
This becomes a problem when an IDE puts its own xml files in a dot-directory inside the DSpace project directory.
In my case, Intellij IDEA created a directory ".idea" in the project base dir and then put some xml file there. 
This seems to cause the "test" step to fail because IDEA's xml file could not be validated.

The problem seems to be related to the wildcards used in the configuration for `xml-maven-plugin`.
Something seems to cause the plugin to "go up a dir" during validation and then enter the .idea folder and fail if it finds a (malformed) xml file in there when only a subset of DSpace's submoduler are selected, e.g. `mvn test --projects dspace,dspace-api,dspace-services`.

```
[ERROR] Failed to execute goal org.codehaus.mojo:xml-maven-plugin:1.2.1:validate (validate-ALL-xml-and-xsl) on project dspace-services: While parsing /home/user/Downloads/DSpace-main/dspace-services/../.idea/foo.xml, at file:/home/user/Downloads/DSpace-main/dspace-services/../.idea/foo.xml, line 1,  column 1: fatal error: Content ist nicht zulässig in Prolog.                                                                                 
[ERROR] While parsing /home/user/Downloads/DSpace-main/dspace-services/../.idea/foo.xml, at file:/home/user/Downloads/DSpace-main/dspace-services/../.idea/foo.xml, line 1,  column 1: fatal error: Content ist nicht zulässig in Prolog.     
```

## To Reproduce
Steps to reproduce the behavior:
1. Checkout DSpace main branch or download zip file of current main
2. cd into DSpace source code folder
3. Create an xml file at `.idea/foo.xml` and put some random content/text in it
4. Run `mvn test` and see it fail  

## Expected behavior
Invalid XML files placed in dot-directories or "hidden" directories should not cause the compilation to fail.

--- a/pom.xml
+++ b/pom.xml
@@ -551,6 +551,7 @@
                             </includes>
                             <excludes>
                                 <exclude>**/target/**</exclude>
+                                <exclude>**/.*/**</exclude>
                             </excludes>
                         </validationSet>
                     </validationSets>

--- a/pom.xml
+++ b/pom.xml
@@ -541,7 +541,7 @@
                 </executions>
                 <configuration>
                     <validationSets>
-                        <!-- validate ALL XML and XSL files throughout the project (excluding target dirs) -->
+                        <!-- validate ALL XML and XSL files throughout the project (excluding target and dot dirs) -->
                         <validationSet>
                             <dir>${root.basedir}</dir>
                             <includes>

